src/bindings.cpp

```diff
@@ -121,6 +128,8 @@ extern "C"
         // The `dartVoiceEndedCallback` is not set on Web.
         if (dartVoiceEndedCallback == nullptr)
             return;
+        if (!isHandleFound)
```

Comment:
If `player` is null, then `isHandleFound` is undefined (i.e. could be anything) at this point, right? Or does C++ set bools to false / `0` at init?